studymanager/src/main/java/io/redlink/more/studymanager/model/StudyImportExport.java

```diff
@@ -19,6 +20,8 @@ public class StudyImportExport {
     private List<Intervention> interventions;
     private Map<Integer, Trigger> triggers;
     private Map<Integer, List<Action>> actions;
+    private Map<String,Integer> participantGroupAssignments;
```

Comment:
Why did you choose `<String, Integer>` and not `<Integer, Integer>` here?

Reply:
As an alternative, we could also export the group-assignment as a simple `List<Integer>`.

For export, you would just do an
```
ParticipantService.listParticipants(studyId)
    .stream()
    .map(Participant::getStudyGroupId)
    .toList()
```

This way we would also keep the assignment-ordering...

Reply:
I couldn't figure out how to make openapi to generate the class in such a way, that the map is a Map<Integer, Integer>. And according to a stackoverflow post, it is not possible, as keys have to be Strings there. But i guess i should have transformed the Strings to Integers inside the Transformer.